packages/server/src/adapters/node-http/content-type/json/getPostBody.ts

```diff
@@ -13,7 +13,8 @@ export async function getPostBody(opts: {
         ok: true,
         data: req.body,
         // If the request headers specifies a content-type, we assume that the body has been preprocessed
-        preprocessed: req.headers['content-type'] === 'application/json',
+        preprocessed:
+          !!req.headers['content-type']?.startsWith('application/json'),
```

Comment:
`startsWith` already returns a `boolean`, no need to coerce.

Reply:
Oh, its possibly `undefined` 😢

Reply:
It's optional chaining, might return undefined. I assumed that's why it was coerced.

Reply:
optional chainging would return undefined, guessing that's why the boolean coercion is there

Reply:
That response speed. Gotta go fast